test/sql/copy/csv/parallel/csv_parallel_clickbench.test_slow

```diff
@@ -125,16 +122,16 @@ CREATE TABLE hits_og
 
 
 statement ok
-INSERT INTO hits_og SELECT * FROM read_parquet('./hits.parquet');
+INSERT INTO hits_og SELECT * FROM read_parquet('https://github.com/duckdb/duckdb-data/releases/download/v1.0/hits.parquet');
 
 statement ok
-COPY hits_og TO '__TEST_DIR__/hits.csv';
+COPY hits_og TO '__TEST_DIR__/hits.csv' (nullstr 'null');
```

Comment:
Are these options necessary? I believe they weren't necessary before, or were they?

Reply:
I'm now sure how this was not a problem before.
I added this because there are empty strings in `SearchPhrase` that would be considered null when the default option of the CSV reader for nullstr is also empty.
I've fixed an issue with `allow_quoted_nulls` on the CSV reader on the rewrite, so maybe that could be it.